src/common/tests/btree.c

```diff
@@ -997,13 +899,100 @@ static struct option btr_ops[] = {
 	{ NULL,		0,			NULL,	0	},
 };
 
+static void
+ts_group(void ** state){
```

Comment:
(style)  "foo ** bar" should be "foo **bar"
(style)  space required before the open brace '{'

Reply:
should just be ts_group(void **state) {